api/apikeys.go

```diff
@@ -0,0 +1,69 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/SkynetLabs/skynet-accounts/jwt"
+	"github.com/julienschmidt/httprouter"
+)
+
+// userAPIKeyPOST creates a new API key for the user.
+func (api *API) userAPIKeyPOST(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+	sub, _, _, err := jwt.TokenFromContext(req.Context())
+	if err != nil {
+		api.WriteError(w, err, http.StatusUnauthorized)
+		return
+	}
+	u, err := api.staticDB.UserBySub(req.Context(), sub, true)
+	if err != nil {
+		api.WriteError(w, err, http.StatusInternalServerError)
+		return
+	}
+	ak, err := api.staticDB.APIKeyCreate(req.Context(), *u)
```

Comment:
Is there a reason we would want to limit this? I'm thinking about how we handle wallet addresses in `skyd`. Instead of blindly creating new keys all the time, try and return an existing key first. 

just a thought, if users are generating keys programmatically, they might add a bunch of bloat to mongo if they accidentally great a new key for every request.

Reply:
I'm not sure we should worry about this at this point. Creating an API key is one more request and you also need to store it somewhere, so that's extra I/O on your end. So, I'm more inclined to think that people would err on the side of creating too few API keys rather than too many. We might get the occasional skapp that would just spam these like crazy but I don't think that's a problem. MongoDB can handle *a lot* of documents in one table.

We can definitely jump on a call/channel and discuss this.

Reply:
F/U: https://github.com/SkynetLabs/skynet-accounts/issues/127